src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -849,7 +850,6 @@ std::string UsgsAstroFrameSensorModel::constructStateFromIsd(const std::string&
     // sun position is not strictly necessary, but is required for getIlluminationDirection.
     state["m_sunPosition"] = getSunPositions(isd);
 
-
     // get sensor_orientation quaternion
     std::vector<double> quaternion = getSensorOrientations(isd, parsingWarnings);
     if (!quaternion.empty() && quaternion.size() != 4) {
```

Comment:
@jessemapel The way this is being parsed now makes the InvalidCamera test for the framer fail via seg fault. Removing the !quaternion.empty check fixes it. Does it matter if the quaternion is empty?